File: old_groovy/src/main/groovy/net/zomis/brainf/model/ast/CommentToken.java
package net.zomis.brainf.model.ast;

public class CommentToken extends Token {

    public final String text;

    public CommentToken(TokenInfo tokenInfo, String text) {
        super(tokenInfo);
        this.text = text;
    }

    @Override
    public String toString() {
        return "CommentToken{" +
                "text='" + text + '\'' +
                "} " + super.toString();
    }

}

File: brainduck-kotlin/src/commonMain/kotlin/net/zomis/brainf/model/ast/CommentToken.kt
package net.zomis.brainf.model.ast;

class CommentToken(tokenInfo: TokenInfo, val text: String) : Token(tokenInfo) {

    override fun toString(): String {
        return "CommentToken{" +
                "text='" + text + '\'' +
                "} " + super.toString();
    }

}
